pkg/operator/util/util.go

```diff
@@ -0,0 +1,34 @@
+package util
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/pkg/operator/api"
+	rookclient "github.com/rook/rook/pkg/rook/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-util")
+var clientTimeout = 15 * time.Second
+
+// GetRookClient gets a reference of the rook client connected to the Rook-API on the given namespace
+func GetRookClient(namespace string, client kubernetes.Interface) (rookclient.RookRestClient, error) {
```

Comment:
Im curious about your reasoning. This is used by operator/cluster and operator/pool. That is why i placed it in operator/util